namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-management-preprod/resources/elasticache.tf

```diff
@@ -0,0 +1,23 @@
+module "ec-cluster-offender-management-allocation-manager" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=3.0"
```

Comment:
Please use the latest version of that module :

See example : https://github.com/ministryofjustice/cloud-platform-environments/pull/995/files

Reply:
Done